src/Bicep.Core/Emit/ExpressionConverter.cs

```diff
@@ -151,11 +152,33 @@ public LanguageExpression ConvertExpression(SyntaxBase expression)
             return (moduleSymbol, propertyAccess.PropertyName.IdentifierName);
         }
 
-        private LanguageExpression GetResourceNameExpression(ResourceSymbol resourceSymbol)
+        public LanguageExpression GetResourceNameExpression(ResourceSymbol resourceSymbol)
```

Comment:
Needed to use this from TemplateWriter. If this is bad layering, please advise.

Reply:
Makes sense to me.

---
In reply to: [577214603](https://github.com/Azure/bicep/pull/1516#discussion_r577214603) [](ancestors = 577214603)